web/tests/waitlist-email-check.test.ts

```
@@ -86,4 +86,46 @@ describe("checkEmailDeliverable", () => {
+      new Promise<never>((_, reject) => {
+        rejecters.push(reject);
+      });
+    resolveMx = hang;
+    resolve4 = hang;
+    resolve6 = hang;
+    const start = Date.now();
+    expect(await checkEmailDeliverable("a@hung.test", 20)).toBe("unknown");
+    expect(Date.now() - start).toBeLessThan(500);
+    // Settle the abandoned lookup so its in-flight entry clears and nothing
+    // lingers for later tests.
+    rejecters.forEach((reject) => reject(new Error("test cleanup")));
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)